pandas/tests/series/test_operators.py

```diff
@@ -91,6 +91,9 @@ def check(series, other, check_reverse=False):
         check(self.ts, 5, check_reverse=True)
         check(tm.makeFloatSeries(), tm.makeFloatSeries(), check_reverse=True)
 
+    def test_pos(self):
+       assert_series_equal(+self.series, 1 * self.series)  #Ensures no rounding issues
+
```

Comment:
this would technically work on datetimes, but I think we should disallow this (pls add tests for ``__neg__`` as well).